### PR TITLE
[MIG] hr_expense_invoice: Migration to 11.0

### PR DESCRIPTION
**Pull request by cubells**
* Standard migration
* convert README.rst to oca fragments

cc @Tecnativa

### CODE REVIEW

**Comment by rafaelbn** [comment]
@cubells are you including this FIX https://github.com/OCA/hr/pull/526 in this PR?

**Comment by cubells** [comment]
@rafaelbn Yes, I am: from this line https://github.com/OCA/hr/pull/525/files#diff-c36127b1752018180a68bd003f15796aR14

**Review by pedrobaeza — APPROVED** [review]

File: hr_expense_invoice/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Supplier invoices on HR expenses',
+    'version': '11.0.1.0.0',
+    'category': 'Human Resources',
+    'author': 'Tecnativa, '
+              'Odoo Community Association (OCA)',
+    'license': 'AGPL-3',
+    'website': 'https://www.tecnativa.com',
```
Comment (nikul-serpentcs) [inline hr_expense_invoice/__manifest__.py]: @cubells Replace URL as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L11)

File: hr_expense_invoice/models/hr_expense_sheet.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+            )
+            if len(c_move_lines) > 1:
+                c_move_lines = c_move_lines[0]
+            residual = line.invoice_id.residual
+            c_move_lines |= line.invoice_id.move_id.line_ids.filtered(
+                lambda x:
+                x.account_id == line.invoice_id.account_id and
+                float_compare(x.credit, residual, precision) == 0)
+            if len(c_move_lines) != 2:
+                raise exceptions.Warning(
```
Comment (nikul-serpentcs) [inline hr_expense_invoice/models/hr_expense_sheet.py]: @cubells use `UserError`

File: hr_expense_invoice/views/hr_expense_views.xml [hunk 3]
```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="hr_expense_form_view" model="ir.ui.view">
+        <field name="name">hr.expense.form</field>
+        <field name="model">hr.expense</field>
+        <field name="inherit_id" ref="hr_expense.hr_expense_form_view"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='product_id']" position="after">
```
Comment (nikul-serpentcs) [inline hr_expense_invoice/views/hr_expense_views.xml]: @cubells IMO Use Direct `field`

**Review by nikul-serpentcs — CHANGES_REQUESTED** [review]
Improve Code

**Comment by cubells** [comment]
@nikul-serpentcs thanks!

done.

**Comment by pedrobaeza** [comment]
Merging as @nikul-serpentcs's comments have been honored.

**Pull request merged**